float32 t_is_activated
float32 t_track_id
float32 active
BoundingBox boxes
float32 t_score
float32 t_label
Loc3D locations_3d